cluster-autoscaler/cloudprovider/cloud_provider.go

```diff
@@ -44,7 +44,10 @@ type NodeGroup interface {
 	// MinSize returns minimum size of the node group.
 	MinSize() int
 
-	// Size returns the current size of the node group.
+	// Size returns the current target size of the node group. It is possible that the
+	// number of nodes in Kubernetes is different at the moment but should be equal
+	// to Size() once everything stabilizes (new nodes finish startup and registration or
+	// removed nodes are deleted completely)
 	Size() (int, error)
```

Comment:
It's usually more convenient for users to have clear method name rather than ambiguous one and 4 lines of semantic description, but as you wish;)